src/bus.ts

```diff
@@ -135,7 +135,7 @@ class MassTransitBus extends EventEmitter implements Bus {
         console.log('Connecting', this.hostAddress.toString());
 
         try {
-            this.connection = connect(this.hostAddress + '?heartbeat=60');
+            this.connection = connect(this.hostAddress + '?heartbeat=60') as unknown as Promise<Connection>;
```

Comment:
how does casting it to unknown then Promise<Connection> help? Why the `as unknown` at all?

Reply:
The bluebird promise type seems to be incompatible with the native Promise type which is circumvented by the `unknown` cast.
When I cast directly to Promise I receive the following error:

> TS2352: Conversion of type 'Bluebird<Connection>' to type 'Promise<Connection>' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.   The types returned by 'then(...)' are incompatible between these types.     Property '[Symbol.toStringTag]' is missing in type 'Bluebird<any>' but required in type 'Promise<any>'.

If you have a better idea I'll gladly change it.